src/components/Navigation.js

```diff
@@ -68,12 +66,7 @@ class Navigation extends React.Component {
               <li className="nav-item">
                 <NavLink to="/" exact className="nav-link" activeClassName="active" activeStyle={{ fontWeight: 'bold' }}>Transactions</NavLink>
               </li>
-              {
-                shouldRenderCustomTokens ?
-                  <li className="nav-item">
-                    <NavLink to="/tokens" exact className="nav-link" activeClassName="active" activeStyle={{ fontWeight: 'bold' }}>Tokens</NavLink>
-                  </li> : null
-              }
+              <ConditionalNavigation to="/tokens" label="Tokens" featureToggle={`explorer-tokens-${REACT_APP_NETWORK}.rollout`} />
```

Comment:
Should we have in the constants file 

`export UNLEASH_BASE_FEATURE_FLAG = explorer-tokens-${REACT_APP_NETWORK}`

Then we can use this everywhere in the code with `UNLEASH_BASE_FEATURE_FLAG.rollout` or `UNLEASH_BASE_FEATURE_FLAG.maintenance`? What do you think?